apps/create-vesting/src/hooks/useGetHooksInfoVestUserAmount.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  TRANSACTION_TYPES,
+  TransactionFactory,
+} from "@bleu/utils/transactionFactory";
+import { useCallback } from "react";
+import { GetHooksTransactionsParams } from "./useGetHooksTransactions";
+import { IHooksInfo, useIFrameContext } from "@bleu/cow-hooks-ui";
+import { scaleToSecondsMapping } from "#/utils/scaleToSecondsMapping";
+import { Address, parseUnits } from "viem";
+import { isERC20Permit } from "#/utils/isERC20Permit";
+
+export const useGetHooksInfoVestUserAmount = () => {
+  const { context, cowShedProxy, jsonRpcProvider } = useIFrameContext();
+
+  return useCallback(
+    async (
+      params: GetHooksTransactionsParams
+    ): Promise<IHooksInfo | undefined> => {
+      const {
+        token,
+        vestingEscrowFactoryAddress,
+        formData: { period, periodScale, amount, recipient },
+      } = params;
+
+      if (!context?.account || !cowShedProxy) return;
+
+      const periodInSeconds = period * scaleToSecondsMapping[periodScale];
+      const amountWei = parseUnits(
```

Comment:
I think that wei is more used as the smallest part of ETH and not any token. I would prefer to use `rawAmount` naming.